signer/src/config/mod.rs

```diff
@@ -247,6 +197,17 @@ impl Validatable for SignerConfig {
             let err = SignerConfigError::NetworkDeployerMismatch;
             return Err(ConfigError::Message(err.to_string()));
         }
+        // At least perform a simple check to see if the database endpoint is
+        // valid for the supported database drivers. We only support PostgreSQL
+        // for now. The rest of the URI we delegate to the database driver for
+        // validation (which will fail fast on startup).
+        if self.db_endpoint.scheme() != "postgresql" {
```

Comment:
Postges supports either `postgresql` or `postgres` as the URI scheme, so we should too.

Reply:
Updated in [d3b79ff](https://github.com/stacks-network/sbtc/pull/521/commits/d3b79ff5626f2537cdb8a4564f11670a13ae156d)